hott-a4-exercises.tex

```diff
@@ -0,0 +1,31 @@
+% -*- mode: LaTeX -*-
+% This is a main LaTeX file for generating the HoTT book
+% for printing on A4 paper. The links are black in this
+% version.
```

Comment:
Do you want to add something to the comment mentioning the one-per-page aspect of this file?